chain-impl-mockchain/src/config.rs

```diff
@@ -93,84 +153,89 @@ mod serde_impl {
 
     impl<'de> Deserialize<'de> for ConfigParam {
```

Comment:
Please, remove this module `serde_impl`. This is never compiled and will never be.